exercises/any_odds_and_evens/any_odds_and_evens_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import "testing"
+
+func TestAnyIsOdd(t *testing.T) {
+	for _, tc := range []struct {
+		s    []int
+		want bool
+	}{
+		{[]int{}, false},
```

Comment:
What about tests with zeros?

Same for tests for other programs, where it's relevant.